whois-api/src/main/java/net/ripe/db/whois/api/rest/search/QueryMessageGenerator.java

```diff
@@ -0,0 +1,21 @@
+package net.ripe.db.whois.api.rest.search;
+
+import com.google.common.collect.ImmutableList;
+import net.ripe.db.whois.api.rest.domain.ObjectMessage;
+import net.ripe.db.whois.api.rest.domain.Parameters;
+import net.ripe.db.whois.common.rpsl.ObjectType;
+import net.ripe.db.whois.common.rpsl.RpslObject;
+
+public interface QueryMessageGenerator {
```

Comment:
QueryMessageGenerator seems like you are generating QueryMessages which are different. May be rename to RpslMessageGenerator and RpslMessage (instead of ObjectMessage). ObjectMessage seems to be way to generic name

Reply:
I will do this change but inside RPSL we have more Messages (Error messages) so I don't know if this is a good name because an error message is an RPSL message as well

Reply:
changed